lib/view/CompressedLayout.abstract.php

```diff
@@ -101,7 +101,7 @@ public static function checkCache($layout, $selectedmodule, $extension) {
 
 		// cache influencers
 		$cache_files = array();
-		$cache_files[] = HTDOCS_PATH . $layout . '/' . ($extension == 'js' ? 'script' : 'style') . '.ini';
+		$cache_files[] = HTDOCS_PATH . "assets/" . $layout . '/' . ($extension == 'js' ? 'script' : 'style') . '.ini';

```

Comment:
PICS_PATH is nog niet outdated, wordt nog gebruikt voor pasfoto's, foto's bij mededelingen en het fotoalbum